system/jlib/jlog.cpp

```diff
@@ -2350,6 +2351,11 @@ void setupContainerizedLogMsgHandler()
     IPropertyTree * logConfig = queryComponentConfig().queryPropTree("logging");
     if (logConfig)
     {
+        if (logConfig->getPropBool(logQueueDisabledAtt, false))
```

Comment:
perhaps should also disable if traceLevel = 0?

Reply:
traceLevel is not something known about at jlib level (but yes, whole area of traceLevel vs loggingDetail is a bit of a mess)